MobileWallet/Screens/Send/AddRecipient/ContactsTableViewController.swift

```diff
@@ -212,7 +217,7 @@ class ContactsTableViewController: UITableViewController {
             return NSLocalizedString("Recent Transactions", comment: "Add recipient view")
         }
 
-        return NSLocalizedString("My Contacts", comment: "Add recipient view")
```

Comment:
@SerhiiShovkoplias chatted to @kukabi about this. Sorry can you keep this "My Contacts".

Reply:
no problem. Ok